app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -214,16 +220,18 @@ export default class CoverageVizBottomSidebar extends React.Component {
     const referenceAccession = (
       <BasicPopup
         trigger={
-          <a
-            href={`https://www.ncbi.nlm.nih.gov/nuccore/${
-              currentAccessionSummary.id
-            }?report=genbank`}
-            target="_blank"
-            rel="noopener noreferrer"
-            className={cs.ncbiLink}
-          >
-            {currentAccessionSummary.id}
-          </a>
+          <div className={cs.ncbiLinkWrapper}>
+            <a
+              href={`https://www.ncbi.nlm.nih.gov/nuccore/${
+                currentAccessionSummary.id
+              }?report=genbank`}
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
I referred to this in another PR. I believe we should create a Link component for this.